src/python_testing/TCP_Tests.py

```diff
@@ -32,8 +32,9 @@
 
 class TCP_Tests(MatterBaseTest):
 
+    # TCP Connection Establishment
     @async_test_body
-    async def test_TCPConnectionEstablishment(self):
+    async def test_TC_SC_8_1(self):
```

Comment:
Entire file should then be named TC_SC_8 at least?

Reply:
I wasn't sure about the naming conventions when I first wrote the script. There are several scripts in `src/python_testing/` with test cases named descriptively, rather than as Ids. While the Id for the test_case helps tally with the test plan, I have added a comment before each test case to describe its purpose. 
From that perspective, I would think leaving the name for the file in its descriptive form helps to easily identify which script contains the TCP tests. WDYT?

Reply:
We currently seem to have most tests named with `TC_SC...` (which I don't particularely like ... because I would find something like `Occupancy` better than OCC) and a few tests that are python tests without actual integration tests ... was wondering if that overlaps.

I guess either is fine. My tendency would have been `when in rome do what romans do` and most romans currently do `TC_....py`.

Reply:
I think the question is what do we want to identify: are these TCP tests or are these `TC_SC` tests. From an external perspective, I find it very hard to say what `SC` stands for (guessing now secure connection ... but yuck). So I like your naming but caution on us being inconsistent.